### PR TITLE
Cohort 1 chapter 12 (geostatistical data): add slides

### PR DESCRIPTION
**Pull request by florisvdh**
Please consider the PRs of @Fgazzelloni first; once those are finished and merged I'll take care of merge conflicts in this PR as needed.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -20,5 +20,6 @@ Imports:
     spData,
     spdep,
     tibble
+    tidyverse
```
Comment (florisvdh) [inline DESCRIPTION]: You forgot the comma here @jonthegeek

File: DESCRIPTION [hunk 2]
```diff
@@ -11,13 +11,15 @@ Depends:
...
     dplyr,
+    geoR,
     ggplot2,
     INLA,
     mapview,
     rmarkdown,
     sf,
     spData,
     spdep,
+    tibble
```
Comment (jonthegeek) [inline DESCRIPTION]:
```suggestion
    tibble,
```

**Comment by jonthegeek** [comment]
Waiting for Federica's PR to finish checks before bothering with checks here.

**Review by jonthegeek — APPROVED** [review]
Federica's stuff is merged. We'll see if this works!

**Pull request merged**